app/Fission/Web/Log/Sentry.hs

```diff
@@ -1,70 +0,0 @@
--- | Error reporting via @sentry.io@
```

Comment:
Actual version is now in `/web`, but somehow GH can't figure that out, soooo 🤷‍♀